### DISCUSSION TITLE
Option to hide tags

### DISCUSSION THREAD

**Discussion by nikecantab** [nathanhoad/godot_dialogue_manager#482] [Ideas]
Loving the tool so far!
One minor nitpick is that I use a LOT of markup tags, so much that reading the actual text is getting difficult. Would it be possible to add an "obstruction free" mode in the editor that only shows the actual text, without any tags or markup?
Something like a visibility icon in the upper right corner you can toggle, not actually changing the text but just hiding stuff while editing.
Again, minor feature and I don't know how much work it would be to add but it would be really appreciated 🙏

**Comment by nathanhoad** [comment]
Unfortunately with the way the `CodeEdit` component works it's not currently possible to partially hide a line of code.

**Reply by nikecantab** [reply]
Got it thanks for explaining 👍
Can you say if it's possible to change the color of the tags to zero their alpha as a hacky way of hiding them? Because if that's the case I'm willing to give it a try to fit it to my own needs myself.
